dali/dfuXRefLib/dfuxreflib.cpp

```diff
@@ -1867,7 +1879,11 @@ class CXRefManager: public CXRefManagerBase
                     Owned<IPropertyTree> results;
                     {
                         CriticalUnblock unblock(manager.logsect);
+#ifdef _CONTAINERIZED
+                        unsigned short port = 0; // No need to connect to Dafs, data is available through mounted storage planes
+#else
                         unsigned short port = getDafsPort(node.endpoint(),numfails,&crit);
+#endif
```

Comment:
I think this should instead check to see if node is localhost, before setting to 0.
i.e. in BM too, if BM, we don't want to go via dafilesrv really. In cloud we *do* want to go via dafilesrv if the file is associated with a non-local hosts. That is not typical, but planes can be configured with hosted storage, so we need to continue to support access via dafielsrv if needed.

Reply:
Fixed to check if the endpoint is localhost.